lib/puppet/ssl/oids.rb

```diff
@@ -84,7 +84,7 @@ def self.register_puppet_oids()
   #  '1.3.6.1.4.1.34380.1.2.1.2':
   #    shortname: 'myothershortname'
   #    longname: 'Other Long name'
-  def self.load_custom_oid_file(custom_oid_file, map_key='oid_mapping')
+  def self.parse_custom_oid_file(custom_oid_file, map_key='oid_mapping')
```

Comment:
I think I'm a little confused by your comment. I essentially broke this into two methods. This one is called `parse_custom_oid_file`, but the `load_custom_oid_file` method below retains the same name and functionality as the previously existing `load_custom_oid_file`. Also, this whole class is marked `@private`.